raygun4js.nuspec

```diff
@@ -2,7 +2,7 @@
 <package xmlns="http://schemas.microsoft.com/packaging/2010/07/nuspec.xsd">
   <metadata>
     <id>raygun4js</id>
-    <version>3.1.2</version>
+    <version>3.1.3</version>
```

Comment:
Probably a minor version, not a patch, but I am not being pedantic here, go as is :-)